src/main/java/org/mule/extensions/jms/internal/config/JmsConfig.java

```diff
@@ -72,6 +74,14 @@
   @ParameterGroup(name = "Producer Config", showInDsl = true)
   private JmsProducerConfig producerConfig;
 
+  /**
+   * Whether to specify a correlationId when publishing messages. This applies both for custom correlation ids specifies at the
+   * operation level and for default correlation Ids taken from the current event
+   */
+  @Parameter
+  @Optional(defaultValue = "AUTO")
+  private OutboundCorrelationStrategy sendCorrelationId = AUTO;
```

Comment:
why `= AUTO`?

Reply:
auto is the intended default behavior

Reply:
I mean: `defaultValue = "AUTO"` should be enough, `sendCorrelationId = AUTO` should not be required.